### DISCUSSION TITLE
Migrate from Mura CMS 7.1 to Masa CMS using Docker

### DISCUSSION THREAD

**Discussion by deeztek** [MasaCMS/MasaCMS#110] [Q&A]
Hi,

I was wondering if there is a migration path from Mura CMS 7.1 to Masa CMS on Docker and if so what would that be?

Thanks

**Comment by guustnieuwenhuis** [comment]
Hi @deeztek,
We use Forgebox to pull in Masa CMS as a dependancy on the building of the Docker image. That way we also control the exact version of Masa CMS that is used.

**Reply by deeztek** [reply]
so you are saying fiddle with the Dockerfile?
